.travis.yml

```diff
@@ -114,6 +118,9 @@ matrix:
       #- env: CCOMPILER='clang-3.8' CXXCOMPILER='clang++-3.8' BUILD_TYPE='Release' BUILD_SHARED_LIBS=ON
 
 before_install:
+  - if [[ ! -z $APT_ARCH ]] ; then sudo dpkg --add-architecture $APT_ARCH ; fi
```

Comment:
If travis will have ARCH parameter it will be nice, because almost all my PRs to include i386 packages into the whitelist were rejected due to duplications. So it is mainly a workaround.
It does not change the main architecture. My local xenial:
```
dpkg --print-architecture
amd64
dpkg --print-foreign-architectures
i386
armhf
```